bom_stock/__openerp__.py

```diff
@@ -20,7 +20,7 @@
 
 {
     'name': 'Compute Stock from BoM',
-    'version': '5.0.2',
+    'version': '8.0.0',
     'category': 'Generic Modules/Others',
     'description':
```

Comment:
remove (in the README file now). 
Adding screenshots in the README file is a must (nice to have)